test_fms/drifters/Makefile.am

```diff
@@ -31,9 +31,9 @@ test_quicksort_SOURCES = test_quicksort.F90
 test_drifters_SOURCES = test_drifters.F90
 
 # Run the test programs.
-TESTS = test_drifters_io.sh test_cloud_interpolator.sh \
-test_drifters_input.sh test_drifters_comm.sh test_drifters_core.sh \
-test_quicksort.sh test_drifters.sh
+TESTS = test_drifters_io.sh test_cloud_interpolator.sh		\
+test_drifters_comm.sh test_drifters_core.sh test_quicksort.sh
+#test_drifters.sh test_drifters_input.sh
```

Comment:
Why are these files commented out?

Reply:
They are not, I just added one and hit the auto re-indent in emacs.